hooks/useCreatePage.ts

```diff
@@ -202,6 +202,10 @@ function useCreatePage({
   const onChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     e.preventDefault();
     const { value } = e.target;
+    const isTitleElement = e.target.dataset.type === "title";
```

Comment:
@xiaoniuniu89 do you think this will work or break anything that I'm not seeing?

Reply:
this onChange handler looks as if it is for creating tags? the onChange for the editor is handled by react hook form. 
Are you sure this is the behaviour you want? I think I am missing something

Reply:
I am thinking you can handle this from the TipTapExstenstions file **or** from the Editor.tsx before calling onChange

Reply:
That's what I assumed too! @MadhavKrishanGoswami would you be able to check if this works as expected and add screenshots?